test/lib/usd/pxrUsdPreviewSurface/testPxrUsdPreviewSurfaceExportImportRoundtrip.py

```diff
@@ -0,0 +1,138 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pxr import Gf
+from pxr import Sdf
+from pxr import Usd
+from pxr import UsdShade
+from pxr import UsdUtils
+
+import os
+import unittest
+
+from maya import cmds
+from maya import standalone
+
+
+class testPxrUsdPreviewSurfaceExport(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        standalone.initialize('usd')
+
+        test_dir = os.path.join(os.path.abspath('.'),
+                                "PxrUsdPreviewSurfaceExportTest")
+
+        cmds.workspace(test_dir, o=True)
+
+    @classmethod
+    def tearDownClass(cls):
+        standalone.uninitialize()
+
+    @unittest.skip("Needs investigation on MacOS")
+    def testPxrUsdPreviewSurfaceRoundtrip(self):
+        """
+        Tests that a pxrUsdPreviewSurface exports and imports correctly.
+        """
+        mayaUsdPluginName = "mayaUsdPlugin"
+        if not cmds.pluginInfo(mayaUsdPluginName, query=True, loaded=True):
+            cmds.loadPlugin(mayaUsdPluginName)
+
+        cmds.file(f=True, new=True)
+
+        sphere_xform = cmds.polySphere()[0]
+
+        material_node = cmds.shadingNode("pxrUsdPreviewSurface", asShader=True)
+
+        material_sg = cmds.sets(renderable=True, noSurfaceShader=True,
+                                empty=True, name=material_node+"SG")
+        cmds.connectAttr(material_node+".outColor",
+                         material_sg+".surfaceShader", force=True)
+        cmds.sets(sphere_xform, e=True, forceElement=material_sg)
+
+        cmds.setAttr(material_node + ".roughness", 0.25)
+        cmds.setAttr(material_node + ".specularColor", 0.125, 0.25, 0.75,
+                     type="double3")
+
+        file_node = cmds.shadingNode("file", asTexture=True,
+                                     isColorManaged=True)
+        uv_node = cmds.shadingNode("place2dTexture", asUtility=True)
+
+        for att_name in (".coverage", ".translateFrame", ".rotateFrame",
+                         ".mirrorU", ".mirrorV", ".stagger", ".wrapU",
+                         ".wrapV", ".repeatUV", ".offset", ".rotateUV",
+                         ".noiseUV", ".vertexUvOne", ".vertexUvTwo",
+                         ".vertexUvThree", ".vertexCameraOne"):
+            cmds.connectAttr(uv_node + att_name, file_node + att_name, f=True)
+
+        cmds.connectAttr(file_node + ".outColor",
+                         material_node + ".diffuseColor", f=True)
+
+        txfile = "PxrUsdPreviewSurfaceExportTest/Brazilian_rosewood_pxr128.png"
```

Comment:
This test should just duplicate the texture map file into its own test directory rather than reach across into the test data of another test. We shouldn't really assume that all of the test data lives in directories right next to each other.